### ISSUE TITLE
Better History Recording

### ISSUE THREAD

**Issue by omodia** [mienaiyami/yomikiru#69]
### Description of the new feature / enhancement

- For example, chapter 1,2,3 are opened, currently they all are recorded in history, instead only record last opened chapter i.e, 3.
- After opening another 2 chapters from the same manga and opened chapter 3 again, then only record chapter 3 in history.
- Only the last opened chapter should be recorded in the history column for that manga.
- Same for different mangas.

### Scenario when this would be used?

- History record will be greatly reduced in this way.
- In this way, each manga will have only one record of history.
- It will be more efficient and the chapter last read from different mangas will be known easily when there large collection of mangas.

![1](https://user-images.githubusercontent.com/116664905/220039896-835f9a04-6e74-4043-93be-6d8a0de16c23.png)

**Comment by mienaiyami** [comment]
> 1px / 2px for the separators in way-2.
In this way even if the bookmarks tab is collapsed/un-collapsed the line end will act as the border.

I didnt understand, can you rephrase.
